js/src/Bars.js

```diff
@@ -603,7 +566,7 @@ var Bars = mark.Mark.extend({
         elements = elements.filter(function(data, index) {
             return indices.indexOf(index) !== -1;
         });
-        elements.selectAll(".bar").style(style);
+        elements.selectAll(".bar").styles(style);
```

Comment:
One of many places where our change from V3 to V5 needs to be handled. 
I believe this change occurred in v4: https://stackoverflow.com/questions/41474452/style-method-does-not-take-object